### ISSUE TITLE
Using Unix domain sockets

### ISSUE THREAD

**Issue by Ctrl-Elite** [datalust/seq-input-healthcheck#43]
Hello,

Is there any way to call Unix domain sockets using `seq-input-healthcheck`?

I'm trying to call docker engine api from inside the SEQ container using `/var/run/docker.sock`.

**Comment by KodrAus** [comment]
Hi @Ctrl-Elite :wave: I haven't tried this myself, but grokking through the source it looks like it should work with `unix:///var/run/docker.sock` since Docker's API is HTTP-based and .NET's HTTP client supports Unix domain sockets. Are you seeing any errors in Seq when you try run the healthcheck?

**Comment by Ctrl-Elite** [comment]
I don't think we can use `unix:///var/run/docker.sock` directly in .NET HTTP client. A `SocketsHttpHandler` is probably needed to achieve this. I can't seem to figure out what should I input in `Target URLs`. I have tried `cURL` URL but that's not working obviously.

**Comment by KodrAus** [comment]
Ah yes it looks like you're right and .NET can't actually use domain sockets by default. If it's something we can support without impacting other usecases then it could be good to support, otherwise your best bet may be to fork the app and make the adjustments you need.
